pkgs/development/python-modules/laszip/default.nix

```diff
@@ -1,48 +1,44 @@
 { lib
+, stdenv
 , buildPythonPackage
 , fetchFromGitHub
 , scikit-build-core
-, distlib
-, pytestCheckHook
-, pyproject-metadata
-, pathspec
 , pybind11
 , cmake
 , LASzip
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "laszip-python";
-  version = "0.2.1";
+  version = "0.2.3";
 
   format = "pyproject";
 
+  disabled = pythonOlder "3.7";
+
   src = fetchFromGitHub {
     owner = "tmontaigu";
     repo = pname;
     rev = version;
-    sha256 = "sha256-ujKoUm2Btu25T7ZrSGqjRc3NR1qqsQU8OwHQDSx8grY=";
+    hash = "sha256-MiPzL9TDCf1xnCv7apwdfcpkFnBRi4PO/atTQxqL8cw=";
   };
 
+  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isLinux "-std=c++17";
```

Comment:
This should probably be `stdenv.cc.isGNU` rather than `stdenv.isLinux` - it's about the compiler being gcc rather than what kernel it's running on.

Reply:
Good catch.
That's right, Thanks!